bin/rspecq

```diff
@@ -38,9 +38,20 @@ OptionParser.new do |o|
 
   o.on("-r", "--redis HOST", "Redis host to connect to " \
        "(default: #{DEFAULT_REDIS_HOST}).") do |v|
+    puts "DEPRECATED: Use --redis-host or --redis-url instead"
```

Comment:
Let's make this more clear about which flag is actually deprecated. So something like:

> --redis is deprecated. Use --redis-host or --redis-url instead